src/Sprite.cpp

```diff
@@ -364,6 +364,8 @@ void Sprite::LoadFromTexture( RageTextureID ID )
 	else
 		pTexture = TEXTUREMAN->LoadTexture( ID );
 
+	if (ActorUtil::GetFileType(ID.filename) == FT_Movie) m_DecodeMovie = true;
```

Comment:
Add braces

Reply:
You right, thank you.
The if-else before that line also doesn't have braces, do I take the opportunity to add them?

Reply:
Yes please!

Reply:
Done!